README.md

```diff
@@ -4,35 +4,63 @@ TavernOfCho
 Install
 ---------
 
+To install TavernOfCho, first clone this repository as following
 ```bash
-docker-compose up -d
+git clone git@github.com:TavernOfCho/config.git TavernOfCho
+# Navigate to the cloned project
+cd TavernOfCho
+# Launch the make init-full command to trigger the installation of the components.
+make init-full
+```
 
-# Website
-docker-compose exec website yarn install
+The last step is to configure properly the ``.env.local`` file from the API and then launch the command ``make composer-dump-dev``.
 
-# API
-docker-compose exec php composer install
-docker-compose exec php php bin/console doctrine:schema:update --force
+Deploying to Production 
+-----------------------
 
-```
+The API is deployed to Heroku.
+
+The Website is deployed to Firebase.
+
+Mercure is deployed to a Dedicated Server as it seems incompatible with Heroku.
+
+Each one has his own process referenced in the .travis.yml file. 
+The process is fully automated and triggered on the master branch.
+
+
+Accessing the project
+---------------------
+
+### In dev
+* Client  : https://127.0.0.1:8050
+* Admin   : https://127.0.0.1:8051
+* API     : https://127.0.0.1:8052
+* Website : https://127.0.0.1:8053
```

Comment:
Website : https://127.0.0.1:8054

Reply:
Resolved in https://github.com/TavernOfCho/config/pull/9/commits/708b54b1d9fc782b3888314f988a7234a1c1aa75